start-tests.sh

```diff
@@ -186,14 +331,18 @@ fi
 
 echo -e
 
-if [ -z "$RECORD" ]; then
+if [[ -z $RECORD ]]; then
   log_color "purple" "RECORD" "not exported; setting to false (set ${PURPLE}RECORD${NC} to true, if you wish to view results within dashboard)\n"
   export RECORD=false
 fi
 
-if [ "$SKIP_UI_TEST" == false ]; then
+# Displaying cypress environment variables, so we know all of the ones that are being passed successfully.
+env | grep "cypress_" -i
+echo -e
```

Comment:
Adding this within the script so we can know exactly what variables were being passed to the cypress tests. It's just an extra layer of logging.